src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@tchaikov @dillaman 
Clang is happy when I keep the reinterpret_cast for `ContextWQ *context_wq = static_cast<ContextWQ *>(wq)` 
And then fix `CompleteContext(0, NULL)` => `CompleteContext(0, static_cast<ContextWQ *>(NULL))
But I'm not yet sure that GNUCC is going to like that.